bin/local-env/docker-compose.yml

```diff
@@ -26,6 +20,8 @@ services:
     volumes:
       - wordpress_data:/var/www/html
       - ../../:/var/www/html/wp-content/plugins/amp
+    env_file:
```

Comment:
Due to how the WP image is now setting the supplied environment variables, the CLI container will no longer be able to read the correct WP constants unless the environment variables are also supplied to that container. The alternative would be to create our own `wp-config.php` file with the preset values and mount it in the WP container.